api-gateway-config/scripts/lua/policies/security.lua

```diff
@@ -24,13 +24,16 @@
 
 local _M = {}
 
+local ngx = require "ngx"
```

Comment:
No need to require `ngx` here because it's already global in OpenResty. Removing this require will also fix the unit test.